include/aikido/util/KinBodyParser.hpp

```diff
@@ -0,0 +1,83 @@
+#ifndef AIKIDO_UTIL_KINBODYPARSER_HPP_
+#define AIKIDO_UTIL_KINBODYPARSER_HPP_
+
+#include <dart/dart.hpp>
+
+namespace aikido {
+namespace util {
+
+/// Read skeleton from a string of OpenRAVE's custom XML format
+///
+/// This function only parses a subset of the format assuming only one body node
+/// in a kinbody file.
+///
+/// Followings are the available fields that this parser can read.
+/// \code
+/// kinbody - attributes: name
+///   body - attributes: name
+///     geom - attributes: name, type* (none, box, sphere, trimesh, cylinder), render
+///       data* (or collision) - file* scale [for trimesh] (see below for the detail)
+///       extents* - 3 float [for box]
+///       height* - float [for cylinder]
+///       radius* - float [for cylinder and sphere]
+///       render - file* scale (see below for the detail)
+/// \endcode
+/// Elements marked with `*` are required ones.
+///
+/// <render>, <data>, or <collision> contains the relative path to a mesh file
+/// and optionally a single float (for the uni-scale) or three float's (for the
+/// x, y, and z-axes) for the scale.
```

Comment:
Nits:
- "contains" -> "contain"
- "uni-scale" -> "for all three axes" (I thought the term uni-scale was a bit confusing, but it's fine to keep this if you